web/pdf_page_view.js

```diff
@@ -113,7 +114,7 @@ class PDFPageView {
     this.paintTask = null;
     this.paintedViewportMap = new WeakMap();
     this.renderingState = RenderingStates.INITIAL;
-    this.resume = null;
+    this.resume = () => {};
```

Comment:
Sorry, but please revert all changes of this kind throughout the patch!

We really don't want to add a, possibly small but nonetheless, overhead to our JavaScript code just for the TypeScript definitions.
Having the TypeScript definitions dictate how we can/cannot write JavaScript code isn't really OK as far as I'm concerned.

Reply:
Sounds good. But this means I'll have to also revert IRenderableView to the below to get the typings to compile:

```
class IRenderableView {
  ...
  /**
   * @type {?(() => void)}
   */
  get resume() {}
}
``` 

Is that okay?

Reply:
No, as was mentioned previously that's also not correct since this isn't a getter.

Reply:
What about something like this? 
```
class IRenderableView {
  constructor() {
    /** @type {(() => void) | null} */
    this.resume = null;
  }
  ...
}
```